README.md

```diff
@@ -3,7 +3,10 @@ Practice Project
 
 Hi World
 
-I am the first of many to come GEDI - Grace Enables Developers of Information. 
+I am the first of many  GEDI - Grace Enables Developers of Information to come. 
+
+My true power is not my many gifts from God. Rather it is my recognition that those gifts from God are not enough for me to realize his plan for me. 
+I need him more than anything else!
```

Comment:
True, you do need God. That is why daily prayer is important - because prayer keeps you constant communication with Him.